src/couch_mrview_http.erl

```diff
@@ -433,6 +443,7 @@ parse_params(Props, Keys) ->
 
 parse_params(Props, Keys, #mrargs{}=Args0) ->
     Args = Args0#mrargs{keys=Keys},
+    [check_compatible(K1, K2, Props) || {K1, K2} <- [{"group", "group_level"}]],
```

Comment:
@kxepal ,  you are right, they are not incompatible at the code level as they both se the group_level field of #mrargs record. But they are incompatible at the user's API level. For example, if user sends group_level=1&group=true, it might not do what they expect ("I want to use grouping, and I want group_level to be 1") because group_level=1 will be ignored.